packages/landing-gatsby/src/templates/processTamplate.js

```diff
@@ -0,0 +1,342 @@
+  return function (elem) {
+    // filter roles, as they don't have a type field
+    if (
+      elem.accountabilities !== null &&
+      componentType.localeCompare('role') === 0
+    ) {
+      return true;
+    }
+    // filter values, tools, policies and structures
+    if (elem.type !== null && elem.type.localeCompare(componentType) === 0) {
+      return true;
+    }
+    return false;
+  };
+};
+
+const BasicTemplate = (props) => {
+  const { pageContext } = props;
+  const { pageContent } = pageContext;
+  const [active, setActive] = useState(0);
+
+  const handleClick = (e) => {
+    const index = parseInt(e.currentTarget.id, 0);
+    if (index !== active) {
+      setActive(index);
+    }
+  };
+  const structures = pageContent.components.filter(byType('structure'));
+  structures.push(
+    ...pageContent.addoptionRequirements.filter(byType('structure'))
+  );
+  const policies = pageContent.components.filter(byType('policy'));
+  const roles = pageContent.components.filter(byType('role'));
+  const tools = pageContent.components.filter(byType('tool'));
+  tools.push(...pageContent.addoptionRequirements.filter(byType('tool')));
+  const values = pageContent.addoptionRequirements.filter(byType('value'));
+  return (
+    <ThemeProvider theme={appTheme}>
+      <Fragment>
+        <SEO title="The best processes for self-managed and distributed teams" />
+        <CookieBanner
+          styles={{
+            banner: { backgroundColor: '#FFDE00' },
+            message: {
+              fontWeight: 500,
+              fontFamily: 'poppins, sans-serif',
+              color: '#091632',
+            },
+          }}
+          message="Yes, we use cookies. To enhance the user experience."
+          onAccept={() => {}}
+          cookie="user-has-accepted-cookies"
+        />
+        <Modal />
+        <ResetCSS />
+        <GlobalStyle />
+        <AppWrapper>
+          <Sun />
+          <Navbar />
+          <Title
+            icon1={pageContent.icons.startIconUrl}
+            icon2={pageContent.icons.endIconUrl}
+            title={pageContent.title}
+            description={pageContent.summary.abstract.substr(0, 180) + '...'}
+          />
+          <Container>
+            <SectionWrapperSmall>
+              <YellowHighliter>
+                <Heading content="Summary" />
+              </YellowHighliter>
+              <MDReactComponent text={pageContent.summary.abstract} />
+            </SectionWrapperSmall>
+            <SectionWrapperSmall>
+              <YellowHighliter>
+                <Heading content="Benefits" />
+              </YellowHighliter>
+              {pageContent.summary.benefits.map((benefit) => (
+                <List text={benefit} />
+              ))}
+            </SectionWrapperSmall>
+            <SectionWrapper>
+              <YellowHighliter>
+                <Heading content="Adoption Requirements" />
+              </YellowHighliter>
+              <dvi>
```

Comment:
Warning: The tag <dvi> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.